dlog/dataLogger/Utils/CMUtils.swift

```diff
@@ -0,0 +1,233 @@
+//
+//  CMUtils.swift
+//  dataLogger WatchKit Extension
+//
+//  Created by Cameron Burroughs on 3/22/22.
+//
+
+import Foundation
+import CoreMotion
+import HealthKit
+
+struct sensorParam {
+    var time: String
+    
+    // gyro values
+    var gyrox: Double
+    var gyroy: Double
+    var gyroz: Double
+    
+    // acc values
+    var accx: Double
+    var accy: Double
+    var accz: Double
+}
+
+class CMUtils: NSObject, HKWorkoutSessionDelegate, HKLiveWorkoutBuilderDelegate {
+
+    let fileUtils = FileUtils()
+    let manager = CMMotionManager()
+    let healthStore = HKHealthStore()
+    let queue = OperationQueue()
+    
+    var WKsession: HKWorkoutSession? = nil
+    var builder: HKLiveWorkoutBuilder? = nil
+    
+    let interval = 1.0/60.0   //sampling interval, may change
+    
+    
+    override init() {}
+    
+    func startWorkoutSession() {
```

Comment:
Write a comment on a piece of code to let me know this needs something.

In general its good to leave a quick one line comment above functions describing their purpose and/or IO
"startWorkoutSession" is self explanatory to an extent, but something like 
" // Fcn communicates with healthstore and generates a workout configuration object to initiate running a background session "
is nice with a little more detail

Reply:
You can add comments by going to the "files Changed" tab at the top and clicking on a plus button to the right of a line number

Reply:
I also know this was my code copied over so I can add in comments too for a lot of this and will sort it up but this is just for reference